dv/uvm/core_ibex/Makefile

```diff
@@ -41,6 +41,9 @@ SIGNATURE_ADDR      := 8ffffffc
 ### Ibex top level parameters ###
 IBEX_CONFIG         := opentitan
 
+# Path to DUT used for coverage reports
+DUT_COV_RTL_PATH		:= "ibex_top"
```

Comment:
One tiny nit, could you swap the tabs for spaces in the Makefile changes please.